docs/circuit_cutting/explanation/index.rst

```diff
@@ -10,39 +10,185 @@ Circuit cutting is a technique to increase the size of circuits we can run on qu
+There are `three settings <https://research.ibm.com/blog/circuit-knitting-with-classical-communication>`__ to consider for circuit cutting.  The first is where only local operations (LO) [i.e., local *quantum* operations] are available.  The other settings introduce classical communication between the circuit executions, which is known in the quantum information literature as LOCC, for `local operations and classical communication <https://en.wikipedia.org/wiki/LOCC>`__.  The LOCC can be either near-time, one-directional communication between the circuit executions (the second setting), or real-time, bi-directional communication (the third setting).
 
-As mentioned above, the cost of any simulation based on quasiprobability distribution is an exponential sampling overhead. The overhead of a cut gate depends on which gate is cut; see the final appendix of [`1 <https://arxiv.org/abs/2205.00016>`__] for details.  Here, we will focus on the CNOT gate.  If no real-time classical communication is available between qubits of the cut gate or wire, cut CNOT gates incur a sampling overhead of O(:math:`9^n`), and wire cuts incur a sampling overhead of O(:math:`16^n`), where :math:`n` is the total number of cuts. If real-time communication is available (i.e., if the hardware supports “dynamic circuits”), the sampling overhead for both CNOT gate and wire cuts may be reduced to O(:math:`4^n`) [`1 <https://arxiv.org/abs/2205.00016>`__,\ `4 <https://arxiv.org/abs/2302.03366>`__]; however, support for circuit cutting with classical communication (LOCC) is not yet supported in CKT.
+As mentioned above, the cost of any simulation based on quasiprobability distribution is an exponential sampling overhead.
+The sampling overhead is the factor by which the overall number of shots must increase for the quasiprobability decomposition to result in the same amount of error, :math:`\epsilon`, as one would get by executing the original circuit.
+The overhead of a cut gate depends on which gate is cut; see the final appendix of [`1 <https://arxiv.org/abs/2205.00016>`__] for details.
+For instance, a single cut CNOT gate incurs a sampling overhead of 9 [`2 <https://arxiv.org/abs/1909.07534>`__,\ `6 <https://arxiv.org/abs/2312.11638>`__].
+A circuit with :math:`n` wire cuts incurs a sampling overhead of O(:math:`16^n`) in the LO setting.  If real-time communication *is* available (i.e., if the hardware supports “dynamic circuits”, also known as the LOCC setting), then the sampling overhead for :math:`n` cut wires may be reduced to O(:math:`4^n`) [`4 <https://arxiv.org/abs/2302.03366>`__].
```

Comment:
I understand that you would need dynamics circuits to implement LOCC but is it accurate to say that dynamic supports and the LOCC setting are the same thing? Perhaps we could rephrase this as: 

```Suggestion
A circuit with :math:`n` wire cuts incurs a sampling overhead of O(:math:`16^n`) in the LO setting. If real-time communication *is* available (i.e., if the hardware supports “dynamic circuits”, which is needed to implement the LOCC setting), then the sampling overhead for :math:`n` cut wires may be reduced to O(:math:`4^n`) [`4 <https://arxiv.org/abs/2302.03366>`__].
```

Reply:
One-way LOCC does not need dynamic circuits.

Two-way LOCC requires dynamic circuits.

I _think_ you can actually get the LOCC wire cutting advantage with only one-way LOCC as long as all the wire cuts move information in the same direction across a circuit separation boundary.

So I should probably rephrase this anyway.

Reply:
Fixed in 81da6cfab8fdbd6a72c422b645d8bde7d27aa90f.